app/src/main/res/layout/state_fragment.xml

```diff
@@ -15,6 +16,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
     <LinearLayout
+      android:id="@+id/linearLayout"
```

Comment:
Please follow ``snake_case`` for layout identifier names for consistency with other identifiers in the codebase.

Ditto elsewhere.

Reply:
Also regarding naming: can we name this something with more context? 'linear layout' is a really generic name that doesn't tell us _what_ should be contained within the layout. The name ``interactionContainer`` below is much clearer since it's indicating that the layout is a container for interactions.

Reply:
followed snake_case naming conventions with clear context